rollup.config.mjs

```diff
@@ -69,10 +72,10 @@ function libBuildOptions({ entrypoints, extension, format, outDir }) {
       tsPlugin({
         exclude: [...testPatterns],
         compilerOptions: {
-          sourceMap: true,
-          inlineSources: true,
+          sourceMap: sourcemap,
+          inlineSources: sourcemap || undefined,
+          removeComments: !sourcemap,
           declaration: false,
-          removeComments: true,
         },
```

Comment:
When compiling the ESM/CJS outputs (.js and .mjs), depending on the `sourcemap: boolean` value, this eithers:

1. generates sourcemaps but strips comments from the generated .js and .mjs outputs;
2. or generates no sourcemaps, and keeps comments in the generated .js and .mjs outputs.

Note that the present configuration only ever calls this function with `{ sourcemap: false }` (second option), so we could hardcode this config as:

```js
tsPlugin({
  compilerOptions: {
    sourceMap: false,
    removeComments: false,
    declaration: false,
  }
})
```

to remove one level of indirection in this config.